test/cmdlineTests/yul_string_format_ascii_long/output.json

```diff
@@ -170,7 +170,9 @@ object \"C_10\" {
             }
 
         }
+
     }
+
```

Comment:
These are added as if I pass emptyString for dependancies to Whiskers it adds new line anyway. Is there a way to avoid this, or it is not important?

Reply:
I think it's not too important